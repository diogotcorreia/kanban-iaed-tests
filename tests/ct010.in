t 1 a
t 2 b
u Pedro
m 0 Pedro IN PROGRESS
m 3 Pedro IN PROGRESS
q
