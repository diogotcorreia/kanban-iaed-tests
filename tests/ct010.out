task 1
task 2
no such task
no such task
